spec.html

```diff
@@ -651,30 +720,32 @@ <h1>%AsyncIterator.prototype%.flatMap ( _mapper_ )</h1>
         </emu-alg>
         <p>The body of %AsyncIterator.prototype%.flatMap is composed of the following steps:</p>
         <emu-alg>
```

Comment:
Suggest to remove the 

```
</emu-alg>
<p>The body of %AsyncIterator.prototype%.flatMap is composed of the following steps:</p>
<emu-alg>
```

to connect them in a single algorithm

Reply:
Whoops, missed that one.